checker/examples/units-extension/FrequencyRelations.java

```diff
@@ -35,11 +41,17 @@ public UnitsRelations init(ProcessingEnvironment env) {
     // Division of a scalar by seconds yields Hertz.
     // Other divisions yield an unannotated value.
     public /*@Nullable*/ AnnotationMirror division(AnnotatedTypeMirror p1, AnnotatedTypeMirror p2) {
-        if (p1.getAnnotations().isEmpty() && p2.getAnnotations().contains(s)) {
+        if (noUnits(p1) && AnnotationUtils.containsSame(p2.getAnnotations(), s)) {
             return hz;
         }
 
         return null;
     }
 
-}
+    // returns true if the type mirror either has no annotations or has only 1 annotation and it is UnknownUnits
+    // any number that isn't assigned a unit will automatically gets the annotation UnknownUnits, eg: int x = 5; //x has @UnknownUnits
+    private boolean noUnits(AnnotatedTypeMirror t) {
```

Comment:
We should not put this in FrequencyRelations, as it gives a bad example and will result in all extensions copying the same code.
Can you put this helper method as a static method in UnitsRelationsDefault?
Then also UnitsAnnotatedTypeFactory can use that method instead of containing a copy.